js/graviex.js

```diff
@@ -0,0 +1,1058 @@
+        };
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = this.marketId (symbol);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit; // default: 500
+        }
+        const response = await this.privateGetTradesHistory (this.extend (request));
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder() requires a symbol argument');
+        }
+        if (side === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder() requires a side argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'market': this.marketId (symbol),
+            'side': side,
+            'volume': type,
+            'price': this.amountToPrecision (symbol, amount),
+        };
+        const response = await this.privatePostOrders (this.extend (request));
+        // {
+        //   "id": 288336422,
+        //   "at": 1644217763,
+        //   "side": "sell",
+        //   "ord_type": "limit",
+        //   "price": "0.141",
+        //   "avg_price": "0.0",
+        //   "state": "wait",
+        //   "market": "ltcbtc",
+        //   "created_at": "2022-02-07T10:09:23+03:00",
+        //   "volume": "0.141",
+        //   "remaining_volume": "0.141",
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "strategy": null
+        // }
+        return this.parseOrder (response);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder() requires a side argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'id': id,
+        };
+        const response = await this.privatePostOrderDelete (this.extend (request, params));
+        // {
+        //   "id": 288336422,
+        //   "at": 1644217763,
+        //   "side": "sell",
+        //   "ord_type": "limit",
+        //   "price": "0.141",
+        //   "avg_price": "0.0",
+        //   "state": "wait",
+        //   "market": "ltcbtc",
+        //   "created_at": "2022-02-07T10:09:23+03:00",
+        //   "volume": "0.141",
+        //   "remaining_volume": "0.141",
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "trades": [],
+        //   "strategy": null
+        // }
+        const order = this.parseOrder (response);
+        const status = order['status'];
+        if (status === 'closed' || status === 'canceled') {
+            throw new OrderNotFound (this.id + ' ' + this.json (order));
+        }
+        return order;
+    }
+
+    async clearOrder (side = undefined, symbol = undefined, params = {}) {
```

Comment:
`clearOrder` is not a standard either, you should rename it to `cancellAllOrders`

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/bb2b0b2ef29c557eee9dad97fee9c010de932c9a